src/routes/+layout.svelte

```diff
@@ -63,17 +64,17 @@
 				{:else if href === '/motd'}
 					<NotebookText class="h-4 w-4" />
 				{/if}
-				{text}
+				{$i18n.t(`sidebar.${text.toLowerCase()}`)}
```

Comment:
Here I don't think you need to call `text.toLowerCase()`, simply update the values in `const SITEMAP` so they are lowercase by default.